moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -1120,9 +1265,10 @@ bool IKFastKinematicsPlugin::getPositionIK(const geometry_msgs::Pose& ik_pose, c
     if (joint_has_limits_vector_[i] && ((ik_seed_state[i] < (joint_min_vector_[i] - LIMIT_TOLERANCE)) ||
                                         (ik_seed_state[i] > (joint_max_vector_[i] + LIMIT_TOLERANCE))))
     {
-      ROS_DEBUG_STREAM_NAMED("ikseed", "Not in limits! " << (int)i << " value " << ik_seed_state[i]
-                                                         << " has limit: " << joint_has_limits_vector_[i] << "  being  "
-                                                         << joint_min_vector_[i] << " to " << joint_max_vector_[i]);
+      ROS_DEBUG_STREAM_NAMED("ikfast", "IK seed not in limits! " << (int)i << " value " << ik_seed_state[i]
```

Comment:
use name_ here

Reply:
``(int)i`` is not required, just use ``i``
``(int)`` is c-style casting, which is bad C++